### PR TITLE
updates the README

### PR DESCRIPTION
**Pull request by npushkarskii**
This PR updates the formatting + changes and/or adds steps to align with the documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,113 @@
...
-6. Change index Configuration :
-   1. Navigate to System > Search and Navigation > Index Configurations. On the right, you can see a list of available configurations. Create or choose an index configuration that you want to run the Coveo search service on
-   2. Navigate to the GENERAL tab.
-   3. Set the search Provider configuration to the one created on step 5
-7. Run the index
-   1. Navigate to System > Search and Navigation > Index Types
-   2. Select an index type and enable existing indexing cronjob
-   3. Before running the cronjob make sure to create the [fields](https://docs.coveo.com/en/2036/index-content/about-fields) and [mapping](https://docs.coveo.com/en/1640/index-content/manage-source-mappings) in Coveo org for the fields you want to index
-   4. Run the cronjob
+1. Create a [Coveo organization](https://docs.coveo.com/en/185)
```
Comment (jsonDaCruzo) [inline README.md:35]: All of the points are numbered with (1.) is it going to appear like that in the read me?

**Review by jsonDaCruzo — APPROVED** [review]
Very good level of detail being added here. Well done @npushkarskii

File: README.md (hunk 1 above)
Comment (npushkarskii) [reply README.md:35]: @jsonDaCruzo  No, that's the convenience of Markdown: you can mark every step as `1.`, but the output would be a perfectly numbered list.
It's very convenient if you need to add/delete steps or swap their places.

You can check how the README renders in this branch:
https://github.com/coveo/searchprovidercoveosearchservices/tree/readme-update

**Review by hdhayneCoveo — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,57 +1,113 @@
...
+
+1. Create a [Catalog Source](https://docs.coveo.com/en/n8of0593)
+
+1. Create an [API key](https://docs.coveo.com/en/1718) with  with the [privilege of editing all sources](https://docs.coveo.com/en/3151/).
+
+   | Service | Privilege | Access level |
+   |---|---|---|
+   | Content | Sources | `Edit all` |
```
Comment (hdhayneCoveo) [inline README.md:43]: same comment i added in the public doc : @npushkarskii 

https://docs.coveo.com/en/n8of0593/coveo-for-commerce/create-a-catalog-source#required-privileges

Comment (npushkarskii) [reply README.md:43]: Thx! Addressed in  #2

**Pull request merged**